common/src/main/java/com/wynntils/features/MountHorseHotkeyFeature.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features;
+
+import com.google.common.collect.ImmutableList;
+import com.wynntils.core.features.Feature;
+import com.wynntils.core.features.properties.FeatureInfo;
+import com.wynntils.core.features.properties.GameplayImpact;
+import com.wynntils.core.features.properties.PerformanceImpact;
+import com.wynntils.core.features.properties.Stability;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.mc.utils.keybinds.KeyHolder;
+import com.wynntils.mc.utils.keybinds.KeyManager;
+import com.wynntils.utils.Delay;
+import com.wynntils.wc.utils.WynnItemMatchers;
+import java.util.List;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.MutableComponent;
+import net.minecraft.network.chat.TranslatableComponent;
+import net.minecraft.network.protocol.game.ServerboundInteractPacket;
+import net.minecraft.network.protocol.game.ServerboundSetCarriedItemPacket;
+import net.minecraft.network.protocol.game.ServerboundUseItemPacket;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.animal.horse.AbstractHorse;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.phys.AABB;
+import org.lwjgl.glfw.GLFW;
+
+@FeatureInfo(stability = Stability.STABLE, gameplay = GameplayImpact.MEDIUM, performance = PerformanceImpact.SMALL)
+public class MountHorseHotkeyFeature extends Feature {
+
+    private static final int searchRadius = 6; // Furthest blocks away from which we can interact with a horse
+    private static final int summonAttempts = 8;
+    private static final int summonDelayTicks = 5;
+    private static int prevItem = -1;
+    private static boolean alreadySetPrevItem = false;
+
+    public enum MountHorseStatus {
+        NO_HORSE("feature.wynntils.mountHorseHotkey.noHorse"),
+        ALREADY_RIDING("feature.wynntils.mountHorseHotkey.alreadyRiding"),
+        ;
```

Comment:
Can't you put the `;` instead of the last `,`?

Reply:
Yea, guess I ignored it cause I thought spotless would clean that up